### DISCUSSION TITLE
「ライセンス...」ボタンについて

### DISCUSSION THREAD

**Discussion by rea-sna** [cocoa-mhlw/cocoa#561] [General]
## その機能リクエストは何らかの問題に関連しますか / Is your feature request related to a problem?
アプリ内設定にある「ライセンス...」ボタンについてです。
iOSではボタンやラベルの文字が見切れると「...」がついてしまうので、「ライセンス」より先にある文字が見切れているのではないかという誤解を招いてしまう。（私もそう思ってました）

## 解決策についてお書きください / Describe the solution you'd like
ボタンを「ライセンス...」から「ライセンス」に変更する。

## その他 / Additional context
該当箇所のスクリーンショットです。
![IMG_1234](https://user-images.githubusercontent.com/47246905/144748204-4643e731-a2b6-4f0a-b9ab-3b5f609f4c21.PNG)

**Comment by keiji** [comment]
ご指摘ありがとうございます。
「右寄せ」も、アクセシビリティの観点から良くないね。みたいな話を開発チームもしていた覚えがあります。

いただいた内容はDiscussionよりIssue（解決すべき課題）として取り扱う方が良いかと思いますので、Issueに移行してはどうかと考えていますがいかがでしょうか。

**Reply by rea-sna** [reply]
ご返信ありがとうございます。
そうだったのですね。この件はIssueに移行させていただきます。

**Reply by keiji** [reply]
再度Issue作成されますか？　こちらから操作してIssueに移行することもできます。
その場合、Issueの作成者がぼく（keiji）になり、rea-snaさんは元のDiscussionの作成者と言う位置づけとなります。

**Reply by rea-sna** [reply]
あ、すみません...ではkeijiさんの方で操作をお願いしても大丈夫ですか...？
（実はこのような大きなプロジェクトに関わらせていただくの初めてで、いろいろ慣れていないのにすみません）

**Reply by keiji** [reply]
#562 に移行したのを確認しました！

**Reply by rea-sna** [reply]
ありがとうございます...！！
お手数をおかけしました ><
